go/protocol/keybase1/extras.go

```diff
@@ -312,6 +312,20 @@ func TeamIDFromString(s string) (TeamID, error) {
 	return TeamID(s), nil
 }
 
+// Used by unit tests.
+func MakeTestTeamID(n uint32) TeamID {
+	b := make([]byte, 8)
+	binary.LittleEndian.PutUint32(b, n)
+	s := hex.EncodeToString(b)
+	c := 2*TEAMID_LEN - len(TEAMID_SUFFIX_HEX) - len(s)
+	s += strings.Repeat("0", c) + TEAMID_SUFFIX_HEX
+	tid, err := TeamIDFromString(s)
+	if err != nil {
+		panic(err)
+	}
+	return tid
+}
```

Comment:
@maxtaco don't you do that for users too?  This is mostly copied from `MakeTestUID`.  KBFS certainly doesn't care about that, and will only use these test IDs in local tests that don't involve the server.